build/Jamfile.v2

```diff
@@ -12,6 +14,7 @@ project boost/test
     : source-location ../src
     : requirements <link>shared:<define>BOOST_TEST_DYN_LINK=1
                    <toolset>borland:<cxxflags>-w-8080
+                   <target-os>cygwin:<define>_POSIX_C_SOURCE=201112L
```

Comment:
This is the same fix I put into the appveyor.yml of all affected repositories that use the boost-ci framework.

https://github.com/jeking3/boost-ci/blob/master/templates/appveyor.yml

The only problem here is that the value should be 200112L and not 201112L

Reply:
So what should I do for the bjam? `200112L` or `20112L` ?